src/main/java/seedu/address/model/event/Description.java

```diff
@@ -16,7 +16,7 @@ public class Description {
      * otherwise " " (a blank string) becomes a valid input.
      * The expression allows for multiple words to be inputted.
      */
-    public static final String VALIDATION_REGEX = "[A-Za-z0-9 _.,!\"'/$]+";
+    public static final String VALIDATION_REGEX = "\\S.*";
```

Comment:
what regex is this?

Reply:
like what does it accept

Reply:
`\\S`: anything non-whitespace
`.`: any character

Reply:
legend